src-tools/netmodel/netmodel.cpp

```diff
@@ -1297,9 +1304,18 @@ void asiotest_udpserv(std::vector<std::string> options) {
 				// [counter] read
 				std::ostringstream oss;
 				oss << "Loop. ";
-                                oss << "Wire: RECV={" << g_speed_wire_recv << "}";
-//				oss << "Wire: RECV={" << g_state_tuntap2wire_in_handler1.get_speed() << "Mbps} ";
-//				oss << "Wire: RECV={" << g_state_tuntap2wire_in_handler2.get_speed() << "Mbps} ";
+
+               // std::size_t find_w = cfg_stats_format.find('w');
+               // std::size_t find_t = cfg_stats_format.find('t');
```

Comment:
this comments can probably be removed now (if not showing important hint for future code)